File: app/src/main/java/com/mifos/passcode/BaseActivity.java
package com.mifos.passcode;

import com.mifos.mobile.passcode.BasePassCodeActivity;

/**
 * Created by dilpreet on 19/01/18.
 */

public class BaseActivity extends BasePassCodeActivity {

    @Override
    public Class getPassCodeClass() {
        //name of the activity which extends MifosPassCodeActivity
        return PassCodeActivity.class;
    }
}

File: app/src/main/java/com/mifos/passcode/BaseActivity.kt
package com.mifos.passcode

import com.mifos.mobile.passcode.BasePassCodeActivity

/**
 * Created by dilpreet on 19/01/18.
 */
class BaseActivity : BasePassCodeActivity() {
    override val passCodeClass: Class<*>
        get() =//name of the activity which extends MifosPassCodeActivity
            PassCodeActivity::class.java
}